js/bytex.js

```diff
@@ -0,0 +1,1455 @@
+        //         "stop": null,
+        //         "fee": 0,
+        //         "fee_coin": "usdt",
+        //         "type": "limit",
+        //         "price": 1.09,
+        //         "status": "new",
+        //         "created_by": 116,
+        //         "created_at": "2021-02-17T02:32:38.910Z",
+        //         "updated_at": "2021-02-17T02:32:38.910Z",
+        //         "User": {
+        //             "id": 116,
+        //             "email": "fight@club.com",
+        //             "username": "narrator",
+        //             "exchange_id": 176
+        //         },
+        //         "fee_structure": {
+        //             "maker": 0.2,
+        //             "taker": 0.2
+        //         },
+        //     }
+        //
+        const marketId = this.safeString (order, 'symbol');
+        const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
+        const meta = this.safeValue (order, 'meta');
+        const filled = this.safeString (order, 'filled');
+        const amount = this.safeString (order, 'size');
+        return this.safeOrder ({
+            'id': this.safeString (order, 'id'),
+            'clientOrderId': undefined,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'status': this.parseOrderStatus (this.safeString (order, 'status')),
+            'symbol': this.safeSymbol (marketId, market, '-'),
+            'type': this.safeString (order, 'type'),
+            'timeInForce': undefined,
+            'postOnly': this.safeValue (meta, 'post_only'),
+            'side': this.safeString (order, 'side'),
+            'price': this.safeString (order, 'price'),
+            'stopPrice': this.safeString (order, 'stop'),
+            'amount': amount,
+            'filled': filled,
+            'remaining': undefined,
+            'cost': undefined,
+            'trades': undefined,
+            'fee': undefined,
+            'average': undefined,
+            'info': order,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        /**
+         * @method
+         * @name bytex#createOrder
+         * @description Create an order on the exchange
+         * @param {str} symbol Unified CCXT market symbol
+         * @param {str} type "limit" or "market" *"market" is contract only*
+         * @param {str} side "buy" or "sell"
+         * @param {float} amount the amount of currency to trade
+         * @param {float} price *ignored in "market" orders* the price at which the order is to be fullfilled at in units of the quote currency
+         * @param {dict} params  Extra parameters specific to the exchange API endpoint
+         * @param {float} params.stopPrice The price at which a trigger order is triggered at
+         * @param {dict} params.meta Object with other options such as post_only
```

Comment:
@ByteX-Dev In bytex's docs for `createOrder`, the param meta has an explanation of `Object with other options such as post_only`. What are some other values that can go inside `meta` (time_in_force maybe?)

![](https://i.imgur.com/1mAwZsB.png)

Reply:
The only option we have available at the moment is post_only.